src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/InternalApi/AxisValueLabelsNormalizer/MetricNormalizer.php

```diff
@@ -40,16 +52,36 @@ public function normalize(ValueInterface $value, string $locale): string
 
         $metric = [
             'amount' => $normalizedMetric['amount']->getData(),
-            'unit' => $value->getUnit()
+            'unit' => $value->getUnit(),
         ];
 
         $localizedMetric = $this->metricLocalizer->localize($metric, $context);
 
-        return sprintf('%s %s', $localizedMetric['amount'], $localizedMetric['unit']);
+        return sprintf(
+            '%s %s',
+            $localizedMetric['amount'],
+            $this->localizeUnit($localizedMetric['unit'], $locale)
+        );
     }
 
     public function supports(string $attributeType): bool
     {
         return AttributeTypes::METRIC === $attributeType;
     }
+
+    /**
+     * @todo @merge master/5.0: rework this method
+     * it should probably use Akeneo\Tool\Bundle\MeasureBundle\Persistence\MeasurementFamilyRepositoryInterface
+     */
+    private function localizeUnit(string $unit, string $locale): string
+    {
+        if (null === $this->translator) {
+            return $unit;
+        }
+
+        $translationKey = sprintf('pim_measure.units.%s', $unit);
+        $translation = $this->translator->trans($translationKey, [], 'messages', $locale);
+
+        return ($translation === $translationKey) ? $unit : $translation;
```

Comment:
Even if it's not configurable from the UI i think it's better to add bracket when there is no translation like all code in the PIM

```suggestion
        return ($translation === $translationKey) ? "[$unit]" : $translation;
```

Reply:
You'd rather have `10 [KILOGRAM]` than `10 KILOGRAM` ?

Reply:
@StevenVAIDIE the former behavior was "10 KILOGRAM", so I prefer letting it like that in the 4.0 branch. 
In master, however, we could add brackets to tip the user that he should add the translation from the UI

Reply:
Yes but maybe i'm wrong ... For example, for Chinese people i think it's more relevant to have this between bracket

Reply:
Ok for me